hypertrace-trace-enricher/hypertrace-trace-enricher-api/src/test/resources/SampleTrace-Hotrod.json

```diff
@@ -0,0 +1,4163 @@
+{
```

Comment:
Curious: Why are we storing this as well? For readability?

Reply:
yes

Reply:
Or, we could have read the structured trace directly from JSON in the code

Reply:
Actually I tried doing that, but while converting to avro it was failing. 
I will move to using the json once I figure out why.